bcda/cli.go

```diff
@@ -156,26 +229,41 @@ func importCCLFDirectory(filePath string) (success, failure, skipped int, err er
 	if err != nil {
 		return 0, 0, 0, err
 	}
-	for _, file := range cclf8 {
-		//todo match cclf0 and validate
-		err = importCCLF8(file)
+
+	for _, file := range cclf0 {
+		aco := file.acoID
+		cclfValidator, err := importCCLF0(file)
 		if err != nil {
-			failure += 1
-			continue
+			log.Error(err)
+			return 0, 0, 0, err
 		}
-		log.Info(fmt.Sprintf("Successfully imported CCLF8 file: %v", file))
-		success += 1
-	}
-	for _, file := range cclf9 {
-		//todo match cclf0 and validate
-		err = importCCLF9(file)
-		if err != nil {
-			failure += 1
-			continue
+
+		for _, file = range cclf8 {
+			if file.acoID == aco {
```

Comment:
If we don't get a match here, then that means that a CCLF8 (or CCLF9) file exists without an accompanying CCLF0 file, correct?  This means that the 8 and/or 9 file will not be imported.  Can we log this scenario?  It would be very hard to troubleshoot that scenario without logging.
To do that, I think we'd have to somehow keep track of files that we imported and compare against total files in the directory :(

Reply:
thinking on this, there's several little things that can happen here.

Reply:
This might be moving into the realm of requirements rather than implementation, but do we know for sure how the files will arrive? We know they will be zipped, but could CCLF0, CCLF8, and CCLF9 be together in one archive? Or is each CCLF file zipped individually? If they're grouped into archives, that could make things easier for us, to confirm that there is a matching set of 0, 8, and 9 together.